Bio/SeqFeature.py

```diff
@@ -941,7 +941,7 @@ def fromstring(text, length=None, circular=False):
             # Attempt to fix features that span the origin
             s_pos = Position.fromstring(s, -1)
             e_pos = Position.fromstring(e)
-            if s_pos > e_pos:
+            if s_pos >= e_pos:
```

Comment:
Will you add the else-clause with ``raise LocationParserError(f"invalid feature location '{text}'")`` as well?

Reply:
Added with https://github.com/biopython/biopython/pull/4526/commits/e3a5ae0f6926e467b667454e0d6e0716c3736f6b

Reply:
Oh - I suppose now that we've fixed the equality edge case, that no longer applies.

Reply:
It seems this messes up some things for alignments, for locations like this `<437..706` that I did not know existed. Not sure how this particular case should be handled

Reply:
Welcome to the joys and murkier corners of GenBank locations. That's a before position, not used much nowadays.

It looks like we should revert e3a5ae0f6926e467b667454e0d6e0716c3736f6b - I was wrong to suggest adding that. This particular bit of code is only trying to find nice and simple origin wrapping locations.